yarn-project/aztec-rpc/src/database/memory_db.ts

```diff
@@ -97,6 +98,45 @@ export class MemoryDB extends MemoryContractDatabase implements Database {
     return Promise.resolve();
   }
 
+  public getGlobalVariablesHash(): Fr {
+    const hash = this.globalVariablesHash;
+    if (!hash) throw new Error(`Global variables hash not set in memory database`);
+    return hash;
+  }
+
+  public setGlobalVariablesHash(hash: Fr) {
+    this.globalVariablesHash = hash;
+    return Promise.resolve();
+  }
+
+  // TODO / Reviewers note: shall i remove the tree roots and globals hash entirely and just have this? Its all the sim needs.
```

Comment:
drawing attention to this comment!

Reply:
Both setters and getters? If we are not using the functions standalone in any cases anymore I think it would be fine to remove it. I don't see many cases where you would only want the `globalVariablesHash` neither, so think the overhead over getting the tree roots directly is fine.